tests/bootstrap.rs

```diff
@@ -29,7 +29,7 @@ async fn test_genesis_node() -> Result<()> {
 
     assert_eq!(pub_key, node.public_key().await);
 
-    assert_next_event!(event_stream, Event::PromotedToElder);
+    assert_next_event!(event_stream, Event::EldersChanged { .. });
```

Comment:
Maybe better to do
```rust
assert_next_event!(event_stream, Event::EldersChanged { change: NodeElderChange::Promoted, .. })
```
to keep the original semantics? Same in the other places.

Reply:
Absolutely, good point.